src/zope/__init__.py

```diff
@@ -1 +0,0 @@
-__import__('pkg_resources').declare_namespace(__name__)  # pragma: no cover
```

Comment:
Would this not also allow `setuptools` to be removed from `install_requires`? I see no other references to `setuptools` or `pkg_resources` in `src`?